client/src/components/landing_page/ExploreMapSection.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <LandingPageSection class='section'
+                      :messages='messages.MapInfo'
+                      :styles='styles.DEFAULT_STYLE'>
+    <img class='static-map'
+         alt=''
+         :src='require(`@/assets/media/static-map.png`)'>
+    <button class='explore-map-button' @click='goToMap'>
+      {{ messages.MapInfo.CTA_BUTTON }}
+    </button>
+  </LandingPageSection>
+</template>
+
+<script lang='ts'>
+import { defineComponent } from 'vue';
+import LandingPageSection from './LandingPageSection.vue';
+import * as messages from '@/assets/messages/landing';
+import { LandingPageStyles as styles } from '@/assets/styles/style_props';
+import { router } from '@/router';
+
+/**
+ * Content for 'Explore map' section of the landing page.
+ */
+export default defineComponent({
+  name: 'ExploreMapSection',
+  components: {
+    LandingPageSection,
+  },
+  data() {
+    return {
+      messages,
+      styles,
+    };
+  },
+  methods: {
+    goToMap(): void {
+      router.push({
+        path: '/map',
+      });
+    },
+  },
+});
+</script>
+
+<style scoped>
+.explore-map-button {
+  width: fit-content;
+  height: 65px;
+  padding: 0 19px;
+  border-radius: 16px;
+  background-color: #FFC300;
+  border: 0;
+  font-size: 18px;
+}
+
+.static-map {
+  width: 920.5px;
+  height: 467.5px;
+  border-radius: 15px;
+
+  background: #FBFBFB;
+  box-shadow: 0 4px 6px 2px rgba(70, 70, 70, 0.25);
+}
+
+.section {
+  padding: 71px 25px;
```

Comment:
This and other pixel values are really specific. The only reason it jumps out at me is generally when this kind of thing happens something has gone awry because there's too many css rules. Generally we want consistent padding on each page

Reply:
This is what mocks specified, which I agree doesn't make sense written out but was trying to stick as close as possible to that. I'll change it.

Reply:
Changed to multiples of 4 (copper has a var for 4px). Let me know if these are still too disparate and I'll try to unify.